### DISCUSSION TITLE
More information about refunds

### DISCUSSION THREAD

**Discussion by beeirl** [etsy/open-api#929] [Q&A]
Is there a way to get information about which part of a shop receipt was (partially) refunded? i.e. transaction, shipping costs etc.

**Comment by SimasBir** [comment]
Any plans on updating the Receipt Refund object with the upcoming Breaking Changes for May 2023 and the addition of "Fully Refunded" and "Partially Refunded" statuses?
Additionally showing split refund amounts with the same receipt amount logic of subtotal/shipping/tax would be great.

**Reply by Gareth064** [reply]
@SimasBir Etsy devs don't monitor this github repo to answer questions like this. You should email them directly with your question developer@etsy.com
